server/routerlicious/packages/services/src/redis.ts

```diff
@@ -44,18 +44,28 @@ export class RedisCache implements ICache {
 	}
 
 	public async get(key: string): Promise<string> {
-		return this.client.get(this.getKey(key));
+		try {
+			return this.client.get(this.getKey(key));
+		} catch (error) {
+			Lumberjack.error(`Error getting ${key} from cache.`, undefined, error);
+			return undefined;
```

Comment:
Can we `throw error` here instead? That will keep the logic path the same

Reply:
Fixed